app/models/blue_sky.rb

```diff
@@ -0,0 +1,16 @@
+BlueSky = DelegateClass(Minisky) do
```

Comment:
Sometimes there's issues with Zeitwerk's reloading and doing something like this but I don't remember what it is exactly. I haven't yet tested it with the app running.